ember/app/models/dependency.js

```diff
@@ -2,5 +2,14 @@ import Model, { attr } from '@ember-data/model';
 
 export default class DependencyModel extends Model {
   @attr name;
-  @attr status;
+  @attr provider;
+
+  get url() {
+    const providerMap = {
+      PIP: 'https://pypi.org/project/',
+      NPM: 'https://npmjs.com/package/',
```

Comment:
move to config if possible

Reply:
as specified in [here](https://github.com/adfinis/Outdated/issues/186) this should also be returned by the backend